pkgs/applications/video/jellyfin-mpv-shim/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub
+, mpv, python-mpv-jsonipc, jellyfin-apiclient-python
+, pillow, tkinter, pystray }:
+
+buildPythonApplication rec {
+  pname = "jellyfin-mpv-shim";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "iwalton3";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0bii85b08fa6sj57x9l8z5rhwlrz36mhf7nck950wzxfgkid2fdv";
+  };
+
+  checkPhase = ''
+    # error: [Errno 13] Permission denied: '/homeless-shelter'
```

comments should try to stay outside of hooks. changing a comment inside of code will cause a rebuild.
```suggestion
  # error: [Errno 13] Permission denied: '/homeless-shelter'
  checkPhase = ''
```

---

moved the comment outside of the hook